microsoft-edge/visual-studio/index.md

```diff
@@ -134,6 +134,48 @@ To attach the Visual Studio debugger to an instance of Microsoft Edge that's alr
 The Visual Studio debugger is now attached to Microsoft Edge.  You can pause the running of JavaScript, set breakpoints, and view `console.log()` statements directly in the **Debug Output** window in Visual Studio.
 
 
+<!-- ====================================================================== -->
+## Edge DevTools extension for Visual Studio
+
+Debug your ASP.NET projects in Visual Studio with the Edge Developer Tools.  You can embed the Microsoft Edge Developer Tools in Microsoft Visual Studio to debug your ASP.NET projects live.  Download the [Microsoft Edge Developer Tools for Visual Studio](https://aka.ms/edgetools-for-vs). 
+
+To use the extension:
+
+1. Make sure Visual Studio 2022 and the ASP.NET workload are installed.
+
+1. Set up Web Live Preview as the default Web Forms Designer, as described in the above page.
+
+1. Open an ASP.NET project.
+
+1. Open a webpage of the project in the **Design** window.
+
+1. In the upper left of the **Design** window, click the **Open Edge DevTools** (![Open Edge DevTools icon.](media/open-edge-dev-tools-v-s-icon.png)) button:
+
+![An ASP.NET project, opening Edge DevTools.](media/devtools-extension-v-s-web-forms-designer.png)
+
+Edge DevTools for Visual Studio opens, with the **Elements** tool selected:
+
+![Microsoft Edge Developer Tools for Visual Studio: Elements tool of DevTools.](media/devtools-extension-visual-studio-elements.png)
+
+The **Network** tool is also open by default:
```

Comment:
Probably replace "open" with "available" as by default, the tab is available but not selected.